packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-input-focus-background-start, 0 0);
+    border: var(--terra-form-input-border, 1px solid #4a4a4a);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    box-shadow: var(--terra-form-input-box-shadow, none);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
```

Comment:
Does opacity of the input change in the consumer theme? Check with @neilpfeiffer on this, I'm thinking if there is an opacity change, it happens at background or color style with rgba instead of changing opacity.

Reply:
I can change this if Neil wants, I just kept this consistent with the `form-textarea` component.

Reply:
checked with @JakeLaCombe - do not add `opacity` here (keep in disabled style block). Too dangerous for someone to misuse that otherwise can already be achieved with rgba values for background, border, color, etc.

He will create an issue to remove it from textarea before it gets used an any other theme.

Reply:
Changed 44e3f71e18905ce93d017024abfc1047c92e6b98